main.py

```diff
@@ -0,0 +1,11 @@
+import os
+from content_creator import ContentCreator
+
+if __name__ == "__main__":
+    QUOTE_API_URL = os.getenv("QUOTE_API_URL")
```

Comment:
How environment variables loaded? Have you created them directly in operation system? You could use `python dotenv` or `django environ` libraries to load envs from .evn file. It is also considered as a good practice to create `.env.example` alongside with script as a blueprint so anybody who wants to use your script could refer to example and fill in.

Reply:
This solution is also suitable here - We gonna use Airflow in future, so part of our credentials would be written in other form